index.yaml

```diff
@@ -46,6 +22,16 @@ distributions:
       ubv8: crystal/release-bionic-arm64-build.yaml
     source_builds:
       default: crystal/source-build.yaml
+  dashing:
+    doc_builds: {}
+    notification_emails:
+      - steven+build.ros2.org@openrobotics.org
+      - ros2-buildfarm-dashing@googlegroups.com
```

Comment:
Please make sure to use the exact same setting on the new Google group as for previous distros (or otherwise mention what was changes and consider applying the same to the other Google groups).

Reply:
The new Dashing group is already set up using side-by-side comparison to match settings with Crystal.

Reply:
I just briefly opened both configuration pages again side-by-side and there are several differences. Please double check them.

Reply:
This has been done.

Reply:
Looks good to me know. I only changed one more setting to make the config the same: disallow posting by all users (except the buildfarm user).